### DISCUSSION TITLE
I have not received my education benefits

### DISCUSSION THREAD

**Discussion by Edison1019** [community/community#139576] [GitHub Education]
### Hi everyone,

I have not received my education benefits within 5 days

**Comment by VadymMakohon** [comment]
Hey.
Did you mean that you applied for Github education pack and did not get your benefits?

**Reply by Edison1019** [reply]
yes, i can't use github copilot

**Comment by VadymMakohon** [comment]
Didi you get approved i mean your documents were accepted, correct?

**Reply by Edison1019** [reply]
yes, i got approved already
![螢幕擷取畫面 2024-09-24 161415](https://github.com/user-attachments/assets/ec180109-aea2-4d5c-a13e-dc3c4421a0c4)

**Comment by VadymMakohon** [comment]
Hmmm, i see. I would also say that i happened to me as well, but in my case the documents were rejected.
I suggest reaching out to GitHub support by opening a ticket for assistance with this issue.
Hope it will help you.

**Reply by Edison1019** [reply]
ok, thanks a lot

**Comment by Injamul2k24** [comment]
A number of people are facing this problem. Contact GitHub support team

**Comment by KhamessiTaha** [comment]
I suggest reaching out to GitHub support and open a ticket for this issue, because apparently alot of people are facing the same issue, if not, it'll be fixed soon.

**Comment by davevad93** [comment]
Hi @Edison1019 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Edison1019 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
